torch/nativert/graph/Graph.cpp

```diff
@@ -0,0 +1,1527 @@
+#include <torch/nativert/graph/Graph.h>
+
+#include <fmt/ostream.h>
+#include <fmt/ranges.h>
+#include <limits>
+#include <queue>
+
+#include <c10/util/Enumerate.h>
+#include <c10/util/FbcodeMaps.h>
+#include <c10/util/StringUtil.h>
+#include <c10/util/string_view.h>
+#include <torch/nativert/executor/Placement.h> // @manual
+#include <torch/nativert/graph/TensorMeta.h> // @manual
+
+namespace torch::nativert {
+
+namespace {
+
+// Workaround for MSVC bug.
+template <typename T, typename U>
+constexpr bool is_same_v = std::is_same_v<T, U>;
+
+bool isBlank(char n) {
+  return isspace(n) || n == '\n';
```

Comment:
std::isspace('\n') is true already https://en.cppreference.com/w/c/string/byte/isspace